File: octofit-tracker/backend/octofit_test_data.js
use octofit_db

db.users.insertMany([
  { email: 'thundergod@mhigh.edu', name: 'thundergod', password: 'thundergodpassword' },
  { email: 'metalgeek@mhigh.edu', name: 'metalgeek', password: 'metalgeekpassword' },
  { email: 'zerocool@mhigh.edu', name: 'zerocool', password: 'zerocoolpassword' },
  { email: 'crashoverride@hmhigh.edu', name: 'crashoverride', password: 'crashoverridepassword' },
  { email: 'sleeptoken@mhigh.edu', name: 'sleeptoken', password: 'sleeptokenpassword' }
]);

db.teams.insertMany([
  { name: 'Blue Team', members: [ 'thundergod@mhigh.edu', 'metalgeek@mhigh.edu', 'zerocool@mhigh.edu', 'crashoverride@hmhigh.edu', 'sleeptoken@mhigh.edu' ] },
  { name: 'Gold Team', members: [ 'thundergod@mhigh.edu', 'metalgeek@mhigh.edu', 'zerocool@mhigh.edu', 'crashoverride@hmhigh.edu', 'sleeptoken@mhigh.edu' ] }
]);

db.activity.insertMany([
  { user: 'thundergod@mhigh.edu', activity_type: 'Cycling', duration: 60, points: 10 },
  { user: 'metalgeek@mhigh.edu', activity_type: 'Crossfit', duration: 120, points: 20 },
  { user: 'zerocool@mhigh.edu', activity_type: 'Running', duration: 90, points: 15 },
  { user: 'crashoverride@hmhigh.edu', activity_type: 'Strength', duration: 30, points: 5 },
  { user: 'sleeptoken@mhigh.edu', activity_type: 'Swimming', duration: 75, points: 12 }
]);

db.leaderboard.insertMany([
  { team: 'Blue Team', score: 100 },
  { team: 'Gold Team', score: 90 }
]);

db.workouts.insertMany([
  { user: 'thundergod@mhigh.edu', workout_type: 'Cycling Training', details: { description: 'Training for a road cycling event' } },
  { user: 'metalgeek@mhigh.edu', workout_type: 'Crossfit', details: { description: 'Training for a crossfit competition' } },
  { user: 'zerocool@mhigh.edu', workout_type: 'Running Training', details: { description: 'Training for a marathon' } },
  { user: 'crashoverride@hmhigh.edu', workout_type: 'Strength Training', details: { description: 'Training for strength' } },
  { user: 'sleeptoken@mhigh.edu', workout_type: 'Swimming Training', details: { description: 'Training for a swimming competition' } }
]);

File: octofit-tracker/backend/octofit_tracker/octofit_test_data.py
# Test data for OctoFit Tracker (Mergington High School)
# This file is for reference only. Actual population is done via the management command.

TEST_USERS = [
    dict(username='thundergod', email='thundergod@mhigh.edu', password='thundergodpassword'),
    dict(username='metalgeek', email='metalgeek@mhigh.edu', password='metalgeekpassword'),
    dict(username='zerocool', email='zerocool@mhigh.edu', password='zerocoolpassword'),
    dict(username='crashoverride', email='crashoverride@hmhigh.edu', password='crashoverridepassword'),
    dict(username='sleeptoken', email='sleeptoken@mhigh.edu', password='sleeptokenpassword'),
]

TEST_TEAMS = [
    dict(name='Blue Team'),
    dict(name='Gold Team'),
]

TEST_ACTIVITIES = [
    dict(activity_type='Cycling', duration=60),
    dict(activity_type='Crossfit', duration=120),
    dict(activity_type='Running', duration=90),
    dict(activity_type='Strength', duration=30),
    dict(activity_type='Swimming', duration=75),
]

TEST_LEADERBOARD = [
    dict(score=100),
    dict(score=90),
    dict(score=95),
    dict(score=85),
    dict(score=80),
]

TEST_WORKOUTS = [
    dict(name='Cycling Training', description='Training for a road cycling event'),
    dict(name='Crossfit', description='Training for a crossfit competition'),
    dict(name='Running Training', description='Training for a marathon'),
    dict(name='Strength Training', description='Training for strength'),
    dict(name='Swimming Training', description='Training for a swimming competition'),
]
